### ISSUE TITLE
How does this work?

### ISSUE THREAD

**Issue by t2rankers** [TangliziGit/ColumnsAnimation#1]
I don't speak Chinese and I don't have any experience in coding so if anybody could tell me how to use this...

**Comment by TangliziGit** [comment]
I'm sorry for seeing this issue too late. Just open animation.html in your browser.

**Comment by t2rankers** [comment]
No, I mean, how does the whole thing work? I don't understand how the whole thing works so I can't understand how to input the data and more.

**Comment by bobwu0214** [comment]
how to change data from data.js to data.csv
